test/runtests.jl

```diff
@@ -80,63 +101,72 @@ end
   Base.cumulative_compile_timing(true)
 end
 
-# Used in both Rings/slpolys.jl and StraightLinePrograms/runtests.jl
-const SLP = Oscar.StraightLinePrograms
+println("Making test list")
 
-include("Aqua.jl")
+testlist = []
```

Comment:
```suggestion
const testlist = []
```

Reply:
Actually, on second thought, perhaps instead of using lots of `push!`, we should just do this:
```
const testlist = [
  "Aqua.jl",

  "printing.jl",

  "PolyhedralGeometry/runtests.jl"
  "Combinatorics/runtests.jl",
  ...
]
```